### PR TITLE
Identity v3 Projects Get

### PR DESCRIPTION
**Pull request by jtopjian**
For #157 
Depends on #163

### CODE REVIEW

**Comment by jtopjian** [comment]
@jrperritt ping for review

**Comment by kaccardi** [comment]
@jtopjian @jrperritt hey, how is this going? I'm anxious for this to get merged. anything I can do to help?

File: openstack/identity/v3/projects/requests.go [hunk 1]
```diff
@@ -50,3 +50,36 @@ func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pa
...
 }
+
+// GetOptsBuilder allows extensions to add additional parameters to
+// the Get request.
+type GetOptsBuilder interface {
+	ToProjectGetQuery() (string, error)
+}
+
+// GetOpts allows you to modify the details included in the Get request.
+type GetOpts struct{}
```
Comment (jrperritt) [inline openstack/identity/v3/projects/requests.go]: Are there options?

Comment (jrperritt) [inline openstack/identity/v3/projects/requests.go]: Nevermind, I just read #157. That seems fine.

Comment (jtopjian) [inline openstack/identity/v3/projects/requests.go]: There _are_, but not implemented. I forget why I didn't implement them. There's a note in the Issue about it:

> #164 does not support the request parameters as described in the API reference and here, though Get has been implemented with GetOpts (and extendible) support for the future.

Comment (jtopjian) [inline openstack/identity/v3/projects/requests.go]: Sounds good - thank you!

**Pull request merged**